packages/sitecore-jss-nextjs/src/index.ts

```diff
@@ -38,6 +38,19 @@ export {
   ComponentFields,
   ComponentParams,
   isServer,
+  LayoutPersonalizationService,
+  PersonalizationResult,
+  PersonalizationLoadResult,
+  LayoutFragmentData,
+  LayoutFragmentService,
+  GraphQLLayoutFragmentService,
+  GraphQLLayoutFragmentServiceConfig,
+  PersonalizationDecisionData,
+  RenderingPersonalizationDecision,
+  PersonalizationDecisionsService,
+  RestPersonalizationDecisionsServiceConfig,
+  RestPersonalizationDecisionsService,
+  LayoutPersonalizationUtils
```

Comment:
Keep only singular nouns the name: LayoutPersonalizationUtil

Reply:
There are several "utils" classes in the JSS, so lets keep it as is